ldap2pg/role.py

```diff
@@ -101,6 +125,30 @@ def alter(self, other):
                     role=other.name, options=other.options)
             )
 
+        if self.config != other.config:
+            for key, value in other.config.items():
+                if other.config[key] != self.config.get(key):
+                    yield Query(
+                        'Set configuration %s=%s of %s.' % (
+                            key, value, other.name,
+                        ),
+                        None,
+                        'ALTER ROLE "%(role)s" SET %(key)s TO %(value)s' %
+                        dict(role=other.name, key=key, value=value)
+                    )
+
+            missing_keys = set(self.config.keys()) - set(other.config.keys())
+            for missing_key in missing_keys:
+                yield Query(
+                    'Reset configuration %s of %s.' % (
+                        missing_key, other.name
+                    ),
+                    None,
+                    'ALTER ROLE "%(role)s" SET %(key)s TO DEFAULT' % dict(
```

Comment:
Ok, there's also the `ALTER ROLE "role" RESET "parameter"`. According to documentation, the two syntaxes have the same effect.